packages/tools/buildServerDocker.ts

```diff
@@ -18,6 +19,14 @@ async function main() {
 	const tagName = argv.tagName;
 	const isPreRelease = getIsPreRelease(tagName);
 	const imageVersion = getVersionFromTag(tagName, isPreRelease);
+	const buildDate = moment(new Date().getTime()).format('YYYY-MM-DDTHH:mm:ssZ');
+	let revision = '';
+	try {
+		revision = await execCommand2('git rev-parse --short HEAD', { showOutput: false });
+	} catch (error) {
+		console.info('not a git repository');
```

Comment:
This kind of log statement is often lost in the middle of many other lines, which makes it hard to know what it means. So it would make sense to put a clearer info message like "Could not get git commit: metadata revision field will be empty".